src/main/java/org/jenkinsci/plugins/configfiles/maven/security/CredentialsHelper.java

```diff
@@ -299,8 +299,9 @@ private static String getServerId(final Node server) {
         NodeList nodes = server.getChildNodes();
         for (int i = 0; i < nodes.getLength(); i++) {
             Node node = nodes.item(i);
-            String name = StringUtils.lowerCase(node.getNodeName());
-            String content = StringUtils.trimToNull(node.getTextContent());
+            String name = node.getNodeName().toLowerCase();
+            final String trimmedTextContent = node.getTextContent().trim();
```

Comment:
Could use `String content = Util.fixEmptyAndTrim(node.getTextContent())` from core instead

Reply:
Where is the `Util` class coming from? I don't really see the benefit of it for one line..

Reply:
`hudson.Util`